src/current/_includes/releases/v24.1/v24.1.0-rc.2.md

```diff
@@ -0,0 +1,26 @@
+## v24.1.0-rc.2
+
+Release Date: May 16, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-0-rc-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that was present since v22.2 where [changefeeds]({% link v24.1/change-data-capture-overview.md %}) with long-running [initial scans]({% link v24.1/create-changefeed.md %}#initial-scan) may incorrectly restore checkpoint job process and drop events during [changefeed restarts]({% link v24.1/changefeed-messages.md %}#duplicate-messages) due to transient errors or node restarts. The bug was most likely to occur in clusters with the following contributing factors:
```

Comment:
```suggestion
- Fixed a bug that was present since v22.2 where [changefeeds]({% link v24.1/change-data-capture-overview.md %}) with long-running [initial scans]({% link v24.1/create-changefeed.md %}#initial-scan) might incorrectly restore checkpoint job progress and drop events during [changefeed restarts]({% link v24.1/changefeed-messages.md %}#duplicate-messages) due to transient errors or node restarts. The bug was most likely to occur in clusters with the following contributing factors:
```

I prefer "might" because "may" is more definitively future tense despite this state (former bug) existing in the past.

I am suggesting process -> progress because this matches the original release note and works better gramatically but if it was changed intentionally and it's more accurate, feel free to ignore this.

Reply:
Cool, oh yeah, thanks for catching "process" is definitely a typo.